<ui version="4.0" >
 <class>muxForm</class>
 <widget class="QDialog" name="muxForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Muxing in progress</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="images.qrc" >
    <normaloff>:/images/icon.png</normaloff>:/images/icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" >
   <property name="margin" >
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="title" >
      <string>Status and progress</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <property name="margin" >
       <number>6</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" >
        <item>
         <widget class="QLabel" name="muxLabel" >
          <property name="text" >
           <string>Muxing in progress.</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="progressLabel" >
          <property name="font" >
           <font>
            <weight>75</weight>
            <bold>true</bold>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="frameShadow" >
           <enum>QFrame::Plain</enum>
          </property>
          <property name="text" >
           <string>Progress: 100.0%</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar" >
        <property name="maximum" >
         <number>1000</number>
        </property>
        <property name="value" >
         <number>0</number>
        </property>
        <property name="textVisible" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Output</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5" >
      <property name="margin" >
       <number>6</number>
      </property>
      <item>
       <widget class="QSplitter" name="splitter" >
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QWidget" name="layoutWidget" >
         <layout class="QVBoxLayout" name="verticalLayout_2" >
          <item>
           <widget class="QLabel" name="label_3" >
            <property name="text" >
             <string>tsMuxeR output:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="stdoutText" >
            <property name="tabChangesFocus" >
             <bool>true</bool>
            </property>
            <property name="undoRedoEnabled" >
             <bool>false</bool>
            </property>
            <property name="readOnly" >
             <bool>true</bool>
            </property>
            <property name="acceptRichText" >
             <bool>false</bool>
            </property>
            <property name="textInteractionFlags" >
             <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget1" >
         <layout class="QVBoxLayout" name="verticalLayout_4" >
          <item>
           <widget class="QLabel" name="label_4" >
            <property name="text" >
             <string>Errors:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="stderrText" >
            <property name="tabChangesFocus" >
             <bool>true</bool>
            </property>
            <property name="undoRedoEnabled" >
             <bool>false</bool>
            </property>
            <property name="readOnly" >
             <bool>true</bool>
            </property>
            <property name="acceptRichText" >
             <bool>false</bool>
            </property>
            <property name="textInteractionFlags" >
             <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <spacer name="horizontalSpacer_2" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okBtn" >
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>25</height>
        </size>
       </property>
       <property name="text" >
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="abortBtn" >
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>25</height>
        </size>
       </property>
       <property name="text" >
        <string>Abort</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc" />
 </resources>
 <connections/>
</ui>
